lib/mapObjects/CRewardableObject.h

```diff
@@ -175,6 +175,12 @@ class DLL_LINKAGE CVisitInfo
 	}
 };
 
+namespace Rewardable
+{
+	const std::array<std::string, 3> SelectModeString{"selectFirst", "selectPlayer", "selectRandom"};
+	const std::array<std::string, 5> VisitModeString{"unlimited", "once", "hero", "bonus", "player"};
+}
+
 /// Base class that can handle granting rewards to visiting heroes.
 /// Inherits from CArmedInstance for proper trasfer of armies
 class DLL_LINKAGE CRewardableObject : public CArmedInstance
```

Comment:
This PR affects a lot! CRewardableObject is inherited by a set of other classes. What does these new settings mean?